product_docs/docs/biganimal/release/getting_started/02_connecting_to_your_cloud/index.mdx

```diff
@@ -5,6 +5,36 @@ redirects:
   - ../02_connect_cloud_account/
 ---
 
-How you connect your cloud to BigAnimal depends on if you purchased BigAnimal from Azure Marketplace or if you purchased directly from EDB and are using your own cloud account to link to BigAnimal. 
+
+## Before connecting your cloud
+
+Before connecting your cloud to BigAnimal, you need to check for the readiness of your cloud account and set permissions to allow maintenance operations. BigAnimal CLI commands are provided to automatically perform the following setup operations: 
+
+### Check for the readiness of your cloud account
+
+* **If connecting to Azure**:
+   - Are the necessary Azure resource providers registered for your subscription?
+   - Is there a restriction on SKUs for the standard Esv3 family and standard D2_v4 VM size?
+   - Is the limit on the number of vCPU and public IP addresses in your region sufficient for your clusters?
+* **If connecting to AWS**:
+   - Is the AWS CLI configured to access your AWS account?
+   - Is the limit on the number of vCPUs and Network Load Balancers (NLBs) in your region sufficient for your clusters?
+
+!!! Important
+    You need to manually increase resource limits or perform required configurations in your cloud, if any of the readiness checks are not met.
+
+
+### Set permissions to allow maintenance operations
+ 
+* **If connecting to AWS**:
```

Comment:
is there a reason the AWS bullet is first here and the Azure bullet is first in the two bullets starting on line 15? If not, I'd use the same order for both lists.

Reply:
Fixed.